src/ProjectTemplates/Web.ProjectTemplates/WebApi-CSharp.csproj.in

```diff
@@ -9,6 +9,8 @@
   <!--#if (OrganizationalAuth || IndividualB2CAuth || EnableOpenAPI) -->
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="${MicrosoftAspNetCoreAuthenticationJwtBearerPackageVersion}" NoWarn="NU1605" />
```

Comment:
When would this be a downgrade? Isn't Identity.Web always referencing an earlier version?